.travis.yml

```diff
@@ -1,11 +1,17 @@
 language: node_js
-node_js: 11
+node_js: 18
+cache:
+  npm: false
+  directories:
+    - "~/.pnpm-store"
+before_install:
+  - curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm@7
+  - pnpm config set store-dir ~/.pnpm-store
+install:
+  - pnpm install
```

Comment:
How about using the official [`pnpm/action-setup`](https://github.com/pnpm/action-setup) instead?

Reply:
I think this is a theoretical problem since I will migrate to Github Actions :)

Reply:
Oops, forgot it Travis, and not GitHub Actions. My bad!